app/assets/stylesheets/application.scss

```diff
@@ -303,3 +310,15 @@ input[type="number"] {
   -moz-appearance: textfield;
   text-align: center;
 }
+
+#country-select-menu {
```

Comment:
I'm not sure I like the name of this. It obviously works right now, but I think it should be named to reflect that this is only for the landing page.